spec/yaml/swiftnav/sbp/system.yaml

```diff
@@ -134,6 +134,83 @@ definitions:
                       - 0: System Healthy
                       - 1: An error has occurred
 
+ - SubSystemReport:
+    short_desc: Sub-system Status report
+    embedded_type: true
+    desc: |
+      Report the general and specific state of a sub-system
+    fields:
+        - component:
+            type: u16
+            desc: Identity of reporting subsystem
+            fields:
+              - 0-15:
+                  desc: Subsystem
+                  values:
+                      - 0: Primary GNSS Antenna
+                      - 1: Measurement Engine
+                      - 2: Corrections Client
+                      - 3: Differential GNSS Engine
+                      - 4: CAN
+                      - 5: Wheel Odometry
+                      - 6: Sensor Fusion Engine
```

Comment:
Is this meant to be covering all the starling subcomponent as well? If so then perhaps some more values - IMU, outputs